src/lang/en/index.ts

```diff
@@ -595,4 +595,21 @@ export default {
     enterPassword: 'Enter password',
     googleOnly: '{Google} accounts only',
   },
+  sbtDetails: {
+    description: 'This SBT provides access to multiple tokens & pools and is attached to your account',
+    issuedBy: '{SBT} issued by',
+    kycVerification: '{KYC} Verification',
+    kycProvider: '{KYC} Provider',
+    accessPermitted: 'Access permitted',
+    listOperable: 'Regulated assets operable',
+    noPermissions: 'No additional permissions',
+    regulatedInsitution: 'Regulated Institution',
+    expiryDate: 'Expiry date',
+    expiryDateTooltip: "The 'Expiry Date' is the deadline for your access to regulated assets of SBT",
+    expiresIn: 'Expires in',
+    expiresInTooltip: "The 'Expires in' shows period or point of time the access is acquired",
+    indefiniteExp: 'open-ended',
+    permission: 'permission',
+    permissions: 'permissions',
```

Comment:
Could we use pluralization here?

https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument

Reply:
I have used this feature in other places on Polkaswap, but here we have number with more font-weight, that means we have to embed <span> on the line and pass it via v-html which wasn't work nice as I remember 
<img width="726" alt="Screenshot 2024-07-31 at 10 47 52" src="https://github.com/user-attachments/assets/22c4709e-7033-4d0a-b273-559059f95a94">